db/migrate/20221014165905_rename_zip_column.rb

```diff
@@ -0,0 +1,5 @@
+class RenameZipColumn < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :tulsa_blotter_arrests, :zip, :city_state_zip
```

Comment:
zip isn't actually provided well from the api so we'll have to parse it out better later